### DISCUSSION TITLE
Can't log in to XRDP - User not known to the underlying authentication module

### DISCUSSION THREAD

**Discussion by StupidRepo** [neutrinolabs/xrdp#2195] [Q&A]
Hey! When I launch the XRDP service, Connect to the machine, and enter my credentials it errors and shows this screen:

<img width="422" alt="Screenshot 2022-03-25 at 7 42 19 AM" src="https://user-images.githubusercontent.com/69256931/160076349-feb3ca99-a41b-4a2b-a60b-b29105446d7e.png">

How do I fix this?

**Comment by matt335672** [comment]
It means the user you're trying to use for PAM authentication is not known to the system.

This isn't a normal user of xrdp - PAM authentication is normally used as another layer of security on top of a proxy configuration

When you connect to the machine, there's a box marked "Session" on the login screen. The value you are setting in there maps to a stanza in `/etc/xrdp/xrdp.ini`. If you post that, we might be able to advise you further.
